pkgs/tools/backup/dedup/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, lz4, snappy, openmp }:
+
+stdenv.mkDerivation rec {
+  pname = "dedup";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://dl.2f30.org/releases/${pname}-${version}.tar.gz";
+    sha256 = "0wd4cnzhqk8l7byp1y16slma6r3i1qglwicwmxirhwdy1m7j5ijy";
+  };
+
+  makeFlags = [
+    "CC:=$(CC)"
+    "PREFIX=${placeholder "out"}"
```

Comment:
What is the advantage of `${placeholder "out"}` to `$out`?

Reply:
I like it because I know it'll work without worrying about quoting/escaping, especially as derivation/builder details change.  At least it does in these simple cases, without requiring examination.

I'm not sure but it may not be appropriate when propagated through multiple scopes or certain things like `passAsFile`-- see #1846.

To me it feels more "direct" than a shell variable reference but I'm not sure how founded that is :).